.github/workflows/actions/auto-request-reviews/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "private": true,
+  "scripts": {
+    "build": "ncc build ./main.ts -o . -m"
+  },
+  "devDependencies": {
+    "@actions/core": "^1.9.1",
+    "@actions/github": "^5.0.3",
+    "@vercel/ncc": "^0.34.0",
+    "typescript": "4.7.4"
+  },
+  "volta": {
+    "node": "12.22.7",
+    "npm": "6.14.15"
+  }
+}
```

Comment:
Can you do me a favor and add this manifest to `.github/dependabot.yml`?

I think we'd want something pretty similar to the root level `package.json`:
```
version: 2
updates:
  # existing config for root level package.json
  - package-ecosystem: 'npm' # See documentation for possible values
    directory: './.github/workflows/actions/auto-request-reviews'
    schedule:
      interval: 'daily'
    open-pull-requests-limit: 50
    # Disable rebasing for pull requests, as having several open pull requests all get simultaneously rebased gets
    # noisy from a notification standpoint
    rebase-strategy: 'disabled'
```

I don't think we need the `ignore` field (AFAIK), since this action will only be running on GHA. We'll just need to rebuild the mini project when we update things like typescript.

Reply:
@rwaskiewicz I think I added this correctly. Obviously let me know if I did it wrong. Not sure how to validate it...